src/u2f.c

```diff
@@ -269,16 +321,31 @@ static void _register(const USB_APDU* apdu, Packet* out_packet)
         return;
     }
 
+    if (!memory_is_initialized()) {
```

Comment:
this part is also the same as in _auth and could be moved to `_unlock_or_show_refresh_screen()`, right?

Reply:
yeah, but I think that reduces the readability of the code. Actually I'm already sceptical about this change since I had to put `async...clear` in a deeper place..